ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+Now navigate to each repository folder that was created, in order to open the folders you will need to type `cd` in front of each folder, though to go back one folder you will need to type `cd ..`
 
 Type following:
 
--	**`cd arctica-greeter`**
+```
+cd arctica-greeter
+```
 
 Now you are in the folder here you will need to run the command:
 
--	**`debuild –uc –us`**
+```
+debuild –uc –us
+```
 
-exit the folder by typing **"`cd ..`"** and navigate to the other folders and do the same
+exit the folder by typing `cd ..` and navigate to the other folders and do the same
 
--	**`cd libpam-x2go`**
--	**`cd lightdm-remote-session-x2go`**
--	**`cd remote-logon-service`**
+```
+cd libpam-x2go
+cd lightdm-remote-session-x2go
+cd remote-logon-service
+```
 
-Go back to the folder where the 4 repositories were created using the command **"`cd ..`"** and install the created packages:
+Go back to the folder where the 4 repositories were created using the command `cd ..` and install the created packages:
 
--	**`sudo dpkg -i *.deb`**
+```
+sudo dpkg -i *.deb
+```
 
 In same folder install thin-client-config-agent:
 
--	**`apt-get install thin-client-config-agent`**
+```
+# apt-get install thin-client-config-agent
+```
 
 ## Stage six 
 
 Open the following file, by typing:
+```
+nano /etc/lightdm/lightdm.conf.d/90-arctica-greeter.conf
+```
+        
+Remove the **"#"** on the following lines:
 
--	**`nano /etc/lightdm/lightdm.conf.d/90-arctica-greeter.conf`**
-	
-Remove the **"`#`"** on the following lines:
-
-- **`#greeter session=arctica-greeter`**
-- **`#greeter-show-manual-login=false`**
+```
+#greeter session=arctica-greeter
+#greeter-show-manual-login=false
+```
 
 So the file will look similar to the picture below.
-![AG-RemoteLogon_fig-06](/ArcticaProject/ArcticaDocs/ArcticaGreeter/img/AG-RemoteLogon_fig-06.png)
+![AG-RemoteLogon_fig-06](./img/AG-RemoteLogon_fig-06.png)
 
-Now press **"`CTRL+O`"**  and press enter to save the file, afterwards exit it by pressing **"`CTRL+X`"**.
+Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.
 
 Now test the Connection to the server and type following:
 
-- **`pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**
-
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"`CTRL+C`"**.
+```
+pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
+```
+If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"CTRL+C"**.
 
 Now edit the environment file, open the file by typing:
 
--	**`nano /etc/environment`**
+```
+nano /etc/environment
```

Comment:
"#" or "$"

There are more occurences of this issue below, please find and fix them yourself.